### PR TITLE
fix: netlify deploy previews

### PR DESCRIPTION
**Pull request by mattrosno**
We have server-side errors in Netlify when going to our library and asset pages, which are Next.js ISR. E.g. https://carbon-platform.netlify.app/assets/carbon-angular

Sharp is a web-app dependency, but we're still getting this error:
image

Also, per Netlify docs, the minimum revalidation time is 60 seconds, so update that from 10. Source: https://github.com/netlify/netlify-plugin-nextjs/blob/main/docs/isr.md#using-isr-on-netlify

### CODE REVIEW

File: netlify.toml [hunk 1]
```diff
@@ -1,2 +1,5 @@
 [[plugins]]
-package = "@netlify/plugin-nextjs"
\ No newline at end of file
+  package = "@netlify/plugin-nextjs"
+
+[functions]
+  external_node_modules = ["sharp"]
```
Comment (jdharvey-ibm) [inline netlify.toml:5]: minor: file has no newline at end of file.

Comment (mattrosno) [reply netlify.toml]: Updated 👍 .

**Comment by mattrosno** [comment]
At a loss at the moment.

Removing `plaiceholder` and `sharp` dependencies and use fixed the server errors and instead, the library and asset details page show 404 page not found.

I tried console.logs in `getStaticProps` but those aren't showing up in Netlify logs.

I temporarily removed Next.js redirects and rewrites. No luck.

This last attempt I tried hardcoding one path so the build recognized ISR with revalidation. No luck there either.

I can't figure out why these routes don't load.

* https://deploy-preview-206--carbon-platform.netlify.app/assets/carbon-charts
* https://deploy-preview-206--carbon-platform.netlify.app/assets/carbon-charts/latest
* https://deploy-preview-206--carbon-platform.netlify.app/assets/carbon-charts/master
* https://deploy-preview-206--carbon-platform.netlify.app/assets/github.ibm.com/matt-rosno/carbon-charts/carbon-charts/master
* https://deploy-preview-206--carbon-platform.netlify.app/assets/github.ibm.com/matt-rosno/carbon-charts/carbon-charts/master/area-simple

**Comment by mattrosno** [comment]
@jdharvey-ibm @alisonjoseph i'm going to close this as the Netlify deploy previews aren't working, and it would be additional work with the addition of our middleware.

**Pull request closed without merging**